taskcluster/ci/signing/kind.yml

```diff
@@ -9,7 +9,7 @@ transforms:
   - taskgraph.transforms.task:transforms
 
 jobs:
-  push:
+  sign:
```

Comment:
Followup: For clarity, it'd be nice to use a `single_dep` loader. This way, you don't have to:

1. Define 1 job per build-type
2. Handle dependencies manually.

You just have a template that you apply to every task that is loaded in this kind.

You can easily filter out based on an attribute, for instance:  https://github.com/mozilla-mobile/fenix/blob/2896b36d14362598a7f8e62e311e54db17991a24/taskcluster/ci/push-apk/kind.yml#L15

Example: https://github.com/mozilla-mobile/fenix/blob/2896b36d14362598a7f8e62e311e54db17991a24/taskcluster/fenix_taskgraph/loader/single_dep.py (this loader should move up to taskgraph, I think)